app/atomic_svc.py

```diff
@@ -271,9 +276,25 @@ async def _prepare_executor(self, test, platform, executor):
         Prepare the command and cleanup, and return them with the needed payloads.
         """
         payloads = []
-
-        command, payloads_command = await self._prepare_cmd(test, platform, executor, test['executor']['command'])
-        cleanup, payloads_cleanup = await self._prepare_cmd(test, platform, executor, test['executor'].get('cleanup_command', ''))
+        dep_construct = ""
+        if 'dependencies' in test:
+            for dependence in test['dependencies']:
+                try:
+                    test_exc = test.get('dependency_executor_name') \
```

Comment:
could this just be `test.get('dependency_executor_name', executor)`?

Reply:
Yep, that seems to be the case. Good catch!